### ISSUE TITLE
Trezor.Sample stuck at Device.ReadAsync in TrezorManagerBase.cs

### ISSUE THREAD

**Issue by landoyjx** [MelbourneDeveloper/Trezor.Net#38]
hi author
I still stuck at Device.ReadAsync in TrezorManagerBase.cs with follow information
device: Trezor One
firmware: latest
I just run Trezor.Net.Sample, the program hang at "Waiting for Trezor... Please plug it in if it is not connected"
I have plugged it in, and when i use trezorctl or Trezor Bridge to check, both of tools works. The device is ok.

In the Trezor.Net.Sample program, i change netcoreapp2.0 to netcoreapp 3.1

**Comment by MelbourneDeveloper** [comment]
Try the develop branch

Are you using Trezor 1, or Model T?

**Comment by landoyjx** [comment]
I am using Trezor 1. I checkout to develop, result is the same.

**Comment by landoyjx** [comment]
When i plug off the device, the sample program throw an IO exception:
Unhandled exception. System.IO.IOException: An error occurred while attempting to read from the device

**Comment by MelbourneDeveloper** [comment]
What is the firmware version?

**Comment by landoyjx** [comment]
it is 1.10.2

**Comment by landoyjx** [comment]
before upgrade to 1.10.2, i also test it with same result

**Comment by MelbourneDeveloper** [comment]
Maybe they made a breaking change...

Turn on all exceptions and try again.

Are you sure it's not just waiting for the pin code?

https://christianfindlay.com/2019/07/14/visual-studio-break-on-all-exceptions/

**Comment by landoyjx** [comment]
No, trezor screen is one lock, no pin input board. In sample code, TrezorManagerBroker have set timeout time. 
with turn on all exception, it is still stuck on, this is the debug video: https://www.youtube.com/watch?v=w2Dso29Opwo

**Comment by MelbourneDeveloper** [comment]
Looks broken. I'll have a look when I've got free time
